packages/server-ct/index.ts

```diff
@@ -0,0 +1,45 @@
+import browsers from '@packages/server/lib/browsers'
+import Project from './src/project-ct'
+
+interface OpenConfig {
+  componentTesting: boolean
+  project: string
+  cwd: string
+  invokedFromCli: boolean
+  config: Record<string, unknown>
+  projectRoot: string
+  browsers: Cypress.Browser[]
+  proxyUrl: string
+  userAgent: boolean
+  socketIoRoute: string
+  chromeWebSecurity: boolean
+  proxyServer?: string
+}
```

Comment:
I believe the CLI options are typed elsewhere... @flotwig said to check
`cli/types/cypress` or `cli/types/npm-module`

Can we use existing types?